examples/test/integrations.test.js

```diff
@@ -0,0 +1,17 @@
+const test = require("ava");
+const { server, app } = require("../integrations/server");
+const port = 4000;
+app.listen({ port });
+const sendQuery = query =>
+  require("./util/sendQuery")(query, server.graphqlPath, port);
+
+test("Sending a request to basic example app should return OK status code and no errors", async t => {
```

Comment:
Looks like this is unfinished? 
Testing hello only?

Reply:
Ah.. But then this kind of test will tell us if API is down. 
I was like - let's use internal REST API inside app, but if we use external API anyway let's test it and remove hello.